rxui-sample-java/src/main/java/com/artemzin/rxui/sample/java/MainPresenter.java

```diff
@@ -59,6 +59,8 @@ Subscription bind(MainView view) {
                 .switchMap(loginAndPassword -> authService.signIn(loginAndPassword.getValue0(), loginAndPassword.getValue1()).subscribeOn(ioScheduler)) // "API request".
                 .share();
 
+        credentials.connect();
```

Comment:
No matter if we merge this PR or not, but resulting subscription should be added to `subscription`